File: Autobase/Frontend/Autobase/src/app/models/dto/response/repairRequest/RepairRequestResponseDto.ts
import { RepairRequestStatus } from "../../../enums/RepairRequestStatus";

export interface RepairRequestResponseDto {
  id: number;
  driverId: number;
  carId: number;
  description: string;
  status: RepairRequestStatus;
  createdAt: string;
  completedAt?: string;        
  updatedAt: string;
}

File: Autobase/Backend/Autobase/src/main/java/com/example/Autobase/dto/response/repairRequest/RepairRequestResponseDto.java
package com.example.Autobase.dto.response.repairRequest;

import lombok.Data;

import java.time.LocalDateTime;


@Data
public class RepairRequestResponseDto {

    private Long id;
    private Long driverId;
    private Long carId;
    private String description;
    private String status;
    private LocalDateTime createdAt;
    private LocalDateTime completedAt;
    private LocalDateTime updatedAt;
}
